portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,19 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['I have a twin sister.','Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function addRandomFact() {
+  const facts =
+      ['I have a twin sister.','I have arachnophobia.','My high school graduating class had only 60 people.',
```

Comment:
Nitpick: add a space between ',' and the following single quote

Reply:
I committed a change which adds a single space between commas and the following single quotes.